tests/sizers/wrapsizer.cpp

```diff
@@ -105,23 +116,54 @@ void WrapSizerTestCase::CalcMin()
     sizeMinExpected.y = wxMax(sizeChild1.y, sizeChild2.y);
 
     wxWindow * const
-        child2 = new wxWindow(m_win, wxID_ANY, wxDefaultPosition, sizeChild2);
+        child2 = new wxWindow(m_leftWin, wxID_ANY, wxDefaultPosition, sizeChild2);
     child2->SetBackgroundColour(*wxYELLOW);
-    m_sizer->Add(child2);
-    m_win->Layout();
+    m_leftWrapSizer->Add(child2);
+    m_leftWin->Layout();
 
-    CPPUNIT_ASSERT_EQUAL( sizeMinExpected, m_sizer->CalcMin() );
+    CHECK( sizeMinExpected == m_leftWrapSizer->CalcMin() );
 
     // Three children will take at least two rows so the minimal size in
     // vertical direction must increase.
     const wxSize sizeChild3 = wxSize(sizeTotal.x/2, sizeTotal.y/5);
     sizeMinExpected.y += sizeChild3.y;
 
     wxWindow * const
-        child3 = new wxWindow(m_win, wxID_ANY, wxDefaultPosition, sizeChild3);
+        child3 = new wxWindow(m_leftWin, wxID_ANY, wxDefaultPosition, sizeChild3);
     child3->SetBackgroundColour(*wxGREEN);
-    m_sizer->Add(child3);
-    m_win->Layout();
+    m_leftWrapSizer->Add(child3);
+    m_leftWin->Layout();
+
+    CHECK( sizeMinExpected == m_leftWrapSizer->CalcMin() );
+}
+
+TEXT_VALIDATOR_TEST_CASE("wxWrapSizer::CalcMinFromMinor", "[wxWrapSizer]")
+{
+    const wxSize rightClientSize = m_rightWin->GetClientSize();
```

Comment:
As we create this window with a fixed, hardcoded client size, we could also use hardcoded sizes here IMO instead of `rightClientSize.x / 4 - 10` etc. This would only make the code more clear.